cli/cmd/config.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 var (
+	cliLogger      = &zap.Logger{}
```

Comment:
having a zeroed reference to a logger, in combination with replacing the pointer later in this file, allows to inject an unconfigured logger if needed on init, but have it configured by runtime. 
Worst case scenario, clients get a zeroed, but valid, logger, instead of a nil pointer